src/lightning_app/runners/cloud.py

```diff
@@ -108,6 +110,7 @@ def dispatch(
                 random_name = "".join(random.choice(string.ascii_lowercase) for _ in range(5))
                 spec = V1LightningworkSpec(
                     build_spec=build_spec,
+                    cluster_id=cluster_id,
```

Comment:
`work.cloud_compute` already has a `clusters` attribute. We should check if there is a conflict between the values there and the `--cluster-id` provided and then either raise or continue with the priority one but that's probably a decision we need from Product

Reply:
discussed internally - for now let's not rely on `work.cloud_compute.clusters` as we don't support the suggested functionality of distributing work between different clusters today